### PR TITLE
enh(UI): Apply new branding for Centreon

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This introduces the new Centreon branding.

https://user-images.githubusercontent.com/12515407/223549429-a323b15a-90f8-4df0-99ec-35fcd0bd7613.mov

Screenshot 2023-03-07 at 14 10 32
Screenshot 2023-03-07 at 21 48 32
Screenshot 2023-03-07 at 21 48 39
Screenshot 2023-03-07 at 21 49 23
Screenshot 2023-03-07 at 21 49 34

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

- The new Centreon logo should be displayed on the login page, the header, the about page and the fallback page

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/Login/index.test.tsx [hunk 1]
```diff
@@ -176,7 +176,7 @@ describe('Login Page', () => {
     mockedAxios.post.mockReset();
   });
 
-  it('displays the login form', async () => {
+  it.only('displays the login form', async () => {
```
Comment (bdauria) [inline centreon/www/front_src/src/Login/index.test.tsx:179]:
```suggestion
  it('displays the login form', async () => {
```

File: centreon/www/front_src/src/Navigation/Sidebar/index.tsx [hunk 2]
```diff
@@ -77,23 +76,22 @@ export interface Props {
 }
 
 export default ({ navigationData }: Props): JSX.Element => {
-  const [open, setOpen] = useState(false);
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
```
Comment (bdauria) [inline centreon/www/front_src/src/Navigation/Sidebar/index.tsx:79]:
```suggestion
  const [isMenuOpen, setIsMenuOpen] = useState(false);
```

File: centreon/www/front_src/src/Navigation/Sidebar/index.tsx [hunk 3]
```diff
@@ -12,7 +12,6 @@ import { Page } from '../models';
 import { headerHeight } from '../../Header';
 
 import Logo from './Logo';
-import MiniLogo from './Logo/LogoMini';
 import NavigationMenu from './Menu';
 
 export const openedDrawerWidth = 165;
```
Comment (bdauria) [inline centreon/www/front_src/src/Navigation/Sidebar/index.tsx:17]:
```suggestion
export const openDrawerWidth = 165;
```

File: centreon/www/front_src/src/ResetPassword/ResetPassword.cypress.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    cy.findByLabelText(labelNewPassword).type('new-password');
+    cy.findByLabelText(labelNewPasswordConfirmation).type('new-password-2');
+    cy.findByLabelText(labelNewPassword).click();
+
+    cy.contains(labelNewPasswordsMustMatch).should('be.visible');
+
+    cy.findByLabelText(labelResetPassword).should('be.disabled');
+  });
+
+  it('redirects the user back to the login page when the page does not have required informations', () => {
```
Comment (bdauria) [inline centreon/www/front_src/src/ResetPassword/ResetPassword.cypress.spec.tsx:122]:
```suggestion
  it('redirects the user back to the login page when the page does not have the required information', () => {

```

**Review by bdauria — COMMENTED** [review]
There is an issue with the generation of the following snapshot:

image

**Review by bdauria — DISMISSED** [review]

**Review by adr-mo — APPROVED** [review]

**Review by paloth — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**